.github/workflows/chrome-store-upload.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
           args: zip -r ./vscd.zip . -i * -x *.git*
 
       - name: Upload to
-        uses: trmcnvn/chrome-addon@v1
+        uses: trmcnvn/chrome-addon@v2
```

Comment:
Hmm, wait.. didn't I already land that? 

https://github.com/igrigorik/videospeed/commit/884be009a45b80a2ea95ac28391c4e2b53e220bd

Reply:
Oops! You're right haha. I'll get this branch deleted